testsuite/MDAnalysisTests/util.py

```diff
@@ -233,6 +233,11 @@ def _warn(self, message, category=None, *args, **kwargs):
         if isinstance(message, Warning):
             self._captured_categories.append(message.__class__)
         else:
+            # as follows Python documentation at
+            # https://docs.python.org/3/library/warnings.html#warnings.warn
+            # if category is None, the default UserWarning is used
+            if category is None:
+                category = UserWarning
```

Comment:
Ah, this is a surprisingly simple fix, very nice. Confirming it also works in my hands.

I'll approve but not merge yet given the release cycle situation.